trucks.js

```diff
@@ -245,10 +269,16 @@ var foodTrucks = [
     schedule: []
   }
 ];
+function filterTrucksByDay(day){
+  day = day[0].toUpperCase() + day.slice[1].toLowerCase();
```

Comment:
You can index into `day` like an array because it's a string. The `slice` method you're calling on day, however, is a function, not an array, so this line of code is just going to cause an error.